edgelet/iotedge/src/check.rs

```diff
@@ -174,133 +195,210 @@ impl Check {
-                        edge_container_version(
-                            "edgeHub",
-                            &check.latest_versions.edge_hub,
-                            check.docker_host_arg.as_ref().map(AsRef::as_ref),
-                        )
-                    }),
                     ("host time is close to real time", host_local_time),
                     ("container time is close to host time", container_local_time),
+                    ("production readiness: certificates", settings_certificates),
+                    (
+                        "production readiness: certificates expiry",
+                        settings_certificates_expiry,
+                    ),
+                    (
+                        "production readiness: container engine",
+                        settings_moby_runtime_uri,
+                    ),
+                    (
+                        "production readiness: logs policy",
+                        container_runtime_logrotate,
+                    ),
+                    ("production readiness: DNS server", container_runtime_dns),
                 ],
             ),
             (
-                "conn",
+                "Connectivity checks",
                 &[
-                    ("can connect to IoT Hub AMQP port", |check| {
-                        connection_to_iot_hub(check, 5671)
+                    (
+                        "host can connect to and perform TLS handshake with IoT Hub AMQP port",
+                        |check| connection_to_iot_hub_host(check, 5671),
+                    ),
+                    (
+                        "host can connect to and perform TLS handshake with IoT Hub HTTPS port",
+                        |check| connection_to_iot_hub_host(check, 443),
+                    ),
+                    (
+                        "host can connect to and perform TLS handshake with IoT Hub MQTT port",
+                        |check| connection_to_iot_hub_host(check, 8883),
+                    ),
+                    ("container can connect to IoT Hub AMQP port", |check| {
+                        connection_to_iot_hub_container(check, 5671)
                     }),
-                    ("can connect to IoT Hub HTTPS port", |check| {
-                        connection_to_iot_hub(check, 443)
+                    ("container can connect to IoT Hub HTTPS port", |check| {
+                        connection_to_iot_hub_container(check, 443)
                     }),
-                    ("can connect to IoT Hub MQTT port", |check| {
-                        connection_to_iot_hub(check, 8883)
+                    ("container can connect to IoT Hub MQTT port", |check| {
+                        connection_to_iot_hub_container(check, 8883)
                     }),
+                    ("edge hub can bind to ports on host", edge_hub_ports_on_host),
+                    (
+                        "container runtime network allows name resolution",
```

Comment:
Is this message correct?

Reply:
Venkat suggests "modules on the container runtime network can resolve each other by name".

Reply:
Oh wait. I thought this was related to the port binding check. You can disregard my comment.